app/javascript/onboarding/Onboarding.jsx

```diff
@@ -1,6 +1,7 @@
 import { h, Component } from 'preact';
 import PropTypes from 'prop-types';
 
+import Focustrap from '../shared/components/focusTrap';
```

Comment:
(nitpick/refactor) - would be nice to have the casing match the component file - not a big deal though, so take or leave this suggestion :) 

```suggestion
import FocusTrap from '../shared/components/focusTrap';
```

Reply:
Just a note about this as we're adding a new package `focus-trap` which is being used in the `<FocusTrap />` component as mentioned in the PR description. It's a new package to the front-end codebase that is [pretty tiny](https://bundlephobia.com/result?p=focus-trap@6.3.0) (2.9kB Minified + Gzipped).

We do not add packages frequently, but I can see the benefit of using this. In terms of affecting page load with more JS, this pain is deferred as the whole onboarding experience is imported dynamically. cc: @benhalpern 

https://github.com/forem/forem/blob/d88baea07730de0a2954ea284ee32e67066b69f2/app/javascript/packs/Onboarding.jsx#L21

Reply:
❤️ @nickytonline I very very much appreciate you beating the drum on this subject. Anything we can do to encourage dynamic imports and be selective in our package choices we're doing it the right way.